test/test_parser.py

```diff
@@ -61,3 +62,18 @@ def test_ephemeris_parser_multiple_file(self):
         # segments from both files
         self.assertTrue(len(orbit_segment.query.all()) == 28)
 
+    def test_find_maximum_distance(self):
```

Comment:
👍

Only concern here is how long this damn test takes to run. But we probably have higher priority things to fix.

Reply:
Added import numpy as np.
test_parser.py currently takes about 53.81 seconds.

Reply:
Yup. :(

I mean, it's fine, I'm just impatient 😝